docs/advanced-topics/legacy/legacy.rst

```diff
@@ -0,0 +1,8 @@
+Legacy Features
+===============
+
+.. toctree::
```

Comment:
Not sure I understand -- Do you want me to insert the contents workflow and tool registration docs themselves inline? I can do that, but it will result in this page becoming the longest single page on our docs by a landslide, and will be difficult for users to navigate.

Reply:
![Screen Shot 2022-03-30 at 9 09 38 AM](https://user-images.githubusercontent.com/1049340/160881154-12d2e145-a58d-4483-9c86-300d444812ab.png)

I meant it's the only page in our doc that I'm aware of that's only links (I could be wrong), so it seems like a bit of an outlier.

We shouldn't block the PR for it and I'm not sure what the solution is.